### PR TITLE
Add a hello message

### PR DESCRIPTION
**Pull request by toshiiw**
This is from a practice session on the contributors summit.

### CODE REVIEW

**Comment by toshiiw** [comment]
usually I guess I must manually add labels. How can I do that?

**Comment by craiglpeters** [comment]
Definitely a lot of networking guys. Wish I knew more!

File: contributors/new-contributor-playground/hej-copenhagen.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Hello from kubecon contributors summit
+
+Nice to see many networking (and openstack!) guys on the same table!
```
Comment (craiglpeters) [inline contributors/new-contributor-playground/hej-copenhagen.md]: This is a nice way to get going

**Pull request closed without merging**